bindings/protocols/mqtt/index.template.html

```diff
@@ -173,9 +188,215 @@ <h3>Possible mappings</h3>
+        command. Remember that the <code>invokeaction</code> operation is mapped to <code>mqv:publish</code>
+        control packet in [[[#default-mappings]]]. 
+      </p>
+      <pre id="example-simple-publish" class="example" title="Read a single coil">
+        {
+            "href": "mqtt://broker.com:1883",
+            "op": [
+                "invokeaction",
+            ], 
+            "mqv:topic": "application/devices/thing1/program/commands/reset",
+        }
+      </pre>
+      <p>
+        Continuing with increasingly more complex examples, [[[#example-property]]] shows how to use the MQTT protocol when handling properties.
+        <code>mqv:retain</code> is used to indicate the ability to read the property rather than just observe it. 
+      </p>
+      <pre id="example-property" class="example" title="Read a single coil">
+        {
+            "href": "mqtt://broker.com:1883",
+            "op": [
+                "readproperty",
+                "writeproperty",
+                "observeproperty"
+            ], 
+            "mqv:retain" : true,
+            "mqv:topic": "application/devices/thing1/properties/test",
+            "mqv:filter": "application/devices/thing1/properties/test",
+        }
+      </pre>
+      <p>
+        If the implementer want to give the ability to WoT clients to conveniently observe all properties in a single call, it is possible
+        to use observerallproperties form in the root of a Thing Description. 
+      </p>
+      <pre id="example-property" class="example" title="Read a single coil">
+              {
+                  "href": "mqtt://broker.com:1883",
+                  "op": [
+                      "observerallproperties"
+                  ], 
+                  "mqv:filter": "application/devices/thing1/properties/#",
+              }
+      </pre>
+      <p>
+        When the remote Web Thing wants to force a minimum level of [[[#qualityofservice]]] for a property, it can use the
+        <code>mqv:qos</code> property in the Form element as shown in [[[#example-qos]]]. This will guarantee that consumers
+        and exposers will get the message at the specified quality of service.
+      </p>
+      <pre id="example-qos" class="example" title="Read a single coil">
+              {
+                  "href": "mqtt://broker.com:1883",
+                  "op": [
+                      "readproperty",
+                      "writeproperty",
+                      "observeproperty"
+                  ], 
+                  "mqv:qos": "qos:1", 
+                  "mqv:retain" : true,
+                  "mqv:topic": "application/devices/thing1/properties/test",
+                  "mqv:filter": "application/devices/thing1/properties/test",
+              }
+     </pre>
+      <p>
+        Finally, [[[#example-full]]] reports a full Thing Description for a commercial device. The broker address is only for
```

Comment:
commercial device is a bit weird to use here. It makes me think of an already existing product